plugins/modules/proxmox_cluster_ha_resources.py

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/python
+
+# Copyright: (c) 2025, Markus Kötter <koetter@cispa.de>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+from __future__ import (absolute_import, division, print_function)
+
+__metaclass__ = type
+
+DOCUMENTATION = r'''
+---
+module: proxmox_cluster_ha_resources
+
+short_description: Management of HA groups in roxmox VE Cluster
+
+version_added: "1.1.0"
+
+description: 
+  - Configure HA groups via /cluster/ha/groups
+
+options:
+    state:
+        description: create or delete
+        required: true
+        choices: ['present', 'absent']
+        type: str
+    name:
+        description: HA resource ID. This consists of a resource type followed by a resource specific name, separated with colon (example: vm:100 / ct:100). For virtual machines and containers, you can simply use the VM or CT id as a shortcut (example: 100). aka sid.
```

Comment:
@felixfontein should the `vm:100` / `ct:100` also be wrapped in a `O( )` or `C( )`?

Reply:
I would use `V(...)` or `C(...)` (it's a value, not an option).

Also `aka` should be changed. Generally https://docs.ansible.com/ansible/devel/dev_guide/style_guide/index.html is a good page on wording; the formatting infos in the docs are a bit hidden, I've created a PR to move it to a separate page: https://github.com/ansible/ansible-documentation/pull/2753